inc/functions/carrier.php

```diff
@@ -6,41 +6,36 @@
  */
 
 use BracketSpace\Notification\Interfaces;
-
+use BracketSpace\Notification\Defaults\Store\Carrier as CarrierStore;
 /**
  * Registers Carrier
  *
- * @uses notification/carriers filter
- * @since  6.0.0
+ * @since  [next] Use Carrier Store.
```

Comment:
We need the `[Next]` tag to be capitalized, otherwise, it won't be replaced.

```suggestion
 * @since  [Next] Use Carrier Store.
```